src/olympia/constants/promoted.py

```diff
@@ -0,0 +1,78 @@
+from django.utils.translation import ugettext_lazy as _
+
+
+class PromotedClass():
+    id = 0
+    name = 'Not Promoted'
+    search_ranking_bump = 0
+    badge = None
+    warning = True
+    pre_review = False
+    admin_review = False
+
+
+class NOT_PROMOTED(PromotedClass):
+    pass
+
+
+class RECOMMENDED(PromotedClass):
+    id = 1
+    name = _('Recommended')
+    search_ranking_bump = 1000  # TODO: confirm this bump
+    badge = 'badge'  # TODO: define what the badge is
```

Comment:
Do you see this being used in an API response, to tell the frontend which badge to display?

Reply:
I don't know at this point - the PRD calls for some groups to have a special badge but I'm not sure if that means the API would be returning a constant indicating a special badge, or if frontend would have the logic to identify a certain group needing a special badge.

Reply:
I suppose no matter what approach we take, it's going to require coordination/synchonization between the server and the frontend. I'm working on the badges right now, and there's a bunch of JS involved, so I don't think the server can return an actual badge to us. Therefore, we'll need to create a mapping between a code returned from the server and a badge type on the frontend. We already do this with all sorts of things, so it's no big deal. 

I guess the question is whether the server needs to understand the concept of a "badge" at all, or if the server can just return a code for the promotion category, and the frontend will understand whether a badge is needed, and if so which badge based on that code. The latter seems simpler from the server perspective, and we'll need some logic on the frontend anyway, so I'd probably take that approach.

Reply:
dropped `badge` in https://github.com/mozilla/addons-server/pull/14898/commits/89826e69b29c8274a2e8c0ee8aa049774c27b41c